src/vs/workbench/contrib/debug/browser/breakpointsView.ts

```diff
@@ -1664,7 +1664,7 @@ registerAction2(class extends Action2 {
 	constructor() {
 		super({
 			id: 'workbench.debug.viewlet.action.toggleBreakpointsActivatedAction',
-			title: localize2('activateBreakpoints', 'Toggle Activate Breakpoints'),
+			title: localize2('activateBreakpoints', 'Toggle Active Breakpoints'),
```

Comment:
I actually think the originally wording is closer to correct, although a little awkward. It is toggling whether all the breakpoints are activated. It's not operating just on the active breakpoints, as your wording makes it sound.

Edge/Chrome devtools use a title that changes "Deactivate Breakpoints"/"Activate Breakpoints".

Maybe "Toggle Breakpoints Activated" would work better for us?

Reply:
I agree with @roblourens. Can we instead add a contextkey to support having a mirror pair of actions whose titles match those of  Chrome devtools? We could also add a second codicon to show at a glance the current button state. I don't think its toggle behaviour is currently clear enough visually.

Reply:
How about "Toggle Active Breakpointing"?

Reply:
Or "Toggle Ignore All Breakpoints"?